.gitignore

```diff
@@ -2,6 +2,7 @@
 /openshift.local.*
 /.project
 /.vagrant
+/assets/nbproject
```

Comment:
its an artifact left by using the netbeans IDE.  Isnt it easier to update the .gitignore file then having to exclude it everytime we make changes.  presumably im the only one using the ide..but it doesnt cost us anything